src/mbgl/gl/renderer_backend.cpp

```diff
@@ -17,8 +17,12 @@ std::unique_ptr<gfx::Context> RendererBackend::createContext() {
     result->enableDebugging();
     result->initializeExtensions(
             std::bind(&RendererBackend::getExtensionFunctionPointer, this, std::placeholders::_1));
+#if defined(__GNUC__) && (__GNUC__ >= 9)
+    return result;
+#else  // defined(__GNUC__) && (__GNUC__ >= 9)
     // Needs move to placate GCC 4.9
     return std::move(result);
```

Comment:
We no longer build for GCC 4.9, minimum is 5.

Reply:
Removed the ifdef, thanks!